### ISSUE TITLE
pre_train model

### ISSUE THREAD

**Issue by chenhuan19871014** [sirius-ai/MobileFaceNet_TF#2]
hey,can you upload your best performance pre_train model?my pc has some problems while runing the training file.

**Comment by chenhuan19871014** [comment]
@xsr-ai  how to save the models which are not included the loss layer.

**Comment by xsr-ai** [comment]
sorry，i can‘t provide the pretrain model.
If you don't want to save the loss layer variables, you can follow that:
MobileFaceNet_vars = [v for v in tf.trainable_variables() if v.name.startswith('MobileFaceNet')]
saver = tf.train.Saver(MobileFaceNet_vars, max_to_keep=args.saver_maxkeep)"

**Comment by sirius-ai** [comment]
pretrained model will be provide later.
close, when need reopen.
